packages/components/addon/components/hds/app-footer/link.hbs

```diff
@@ -0,0 +1,25 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::AppFooter::Item>
+  {{! Q: Use Hds::Interactive for link or just Link::Inline? }}
+  <Hds::Link::Inline
+    class="hds-app-footer__link"
+    @color="secondary"
+    @route={{@route}}
+    @models={{hds-link-to-models @model @models}}
+    @query={{hds-link-to-query @query}}
+    @current-when={{@current-when}}
+    @replace={{@replace}}
+    @isRouteExternal={{@isRouteExternal}}
+    @href={{@href}}
+    @isHrefExternal={{@isHrefExternal}}
+    @icon={{@icon}}
+    @iconPosition={{@iconPosition}}
```

Comment:
According to latest design decisions (see comments in Figma) we should not expose this.

Reply:
I commented in the thread that we _could_ expose them for the Ember component.

@andgen404 @heatherlarsen tell me what I should do. Delete these or keep them?

Reply:
I'm inclined to keep these icon props as I don't see what it hurts but let me know if you have a strong opinion otherwise.

@didoo @andgen404

Reply:
I'm resolving this just because it's old and I don't know if it's relevant any longer. Open a new comment if needed.